build/cmake/CopyImportedTargetProperties.cmake

```diff
@@ -0,0 +1,88 @@
+#.rst:
```

Comment:
This CMake script is needed to duplicate an imported library (like `GLEW::glew`) under a different name (`GLEW::GLEW`)
Unfortunately the simple solution `add_library(GLEW:GLEW ALIAS GLEW::glew)` does not work.